AppSettingsManagerAPI/AppSettingsManagerApi.Infrastructure/MySql/Converters/ServiceConfiguration.cs

```diff
@@ -7,18 +7,18 @@ public static class ServiceConfiguration
 {
     public static IServiceCollection AddConverters(this IServiceCollection services)
     {
-        services.AddSingleton<IBidirectionalConverter<Model.User, User>, UserBiverter>();
+        services.AddSingleton<IBidirectionalConverter<Model.User, User>, UserConverter>();
```

Comment:
Nice, don't even need the interface to be bidirectional probably. Don't think we ever actually go from model version to mysql version